src/loops/homotheties.lean

```diff
@@ -0,0 +1,58 @@
+import linear_algebra.affine_space.affine_map
+import analysis.normed_space.basic
+
+open affine_map
+
+-- TODO Drop this lemma once bump Mathlib since it contains
+-- https://github.com/leanprover-community/mathlib/pull/9360
+@[simp] lemma affine_map.homothety_apply_same
+  {k V P : Type*} [comm_ring k] [add_comm_group V] [module k V] [add_torsor V P]
+  (c : P) (r : k) : homothety c r c = c :=
+line_map_same_apply c r
+
+variables {E : Type*} [normed_group E] [normed_space ℝ E]
+
+lemma exists_homothety_mem_of_mem_interior (x : E) {s : set E} {y : E} (hy : y ∈ interior s) :
+  ∃ ε > (0 : ℝ), ∀ (δ : ℝ), ∥δ∥ < ε → homothety x (1 + δ) y ∈ s :=
```

Comment:
This `∥δ∥` feels a bit weird.  Would you mind replacing it by `abs δ`. You could introduce a local notation for `abs` (as is done in the tutorial project). Replacing the `ε.norm_eq_abs` in the last `simp` by `real.norm_eq_abs` is enough to fix the proof.

Reply:
Agreed; now done.